src/components/button/button.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from '@emotion/styled'
+
+export const ButtonStyled = styled.button`
+  width: 375px;
+  height: 58px;
+  margin-top: 20px;
+  margin-left: 20px;
```

Comment:
we rather do not want to have margins on component itself. that's "parent" job to place them correctly in container